test/__tests__/data/tiktok_ads_input.json

```diff
@@ -2353,6 +2355,7 @@
         "eventId": "1616318632825_357",
         "products": [
           {
+            "contentType": "product_group",
```

Comment:
No change in output payload generated?

Reply:
Yes I have updated the input accordingly.